contracts/test/v0.8/foundry/vrf/VRFV2PlusWrapper.t.sol

```diff
@@ -136,28 +136,14 @@ contract VRFV2PlusWrapperTest is BaseTest {
     new VRFV2PlusWrapper(address(0), address(0), address(0));
   }
 
-  function testSetLinkAndLinkNativeFeed() public {
-    VRFV2PlusWrapper wrapper = new VRFV2PlusWrapper(address(0), address(0), address(s_testCoordinator));
+  function testSetLinkNativeFeed() public {
+    VRFV2PlusWrapper wrapper = new VRFV2PlusWrapper(address(s_linkToken), address(0), address(s_testCoordinator));
```

Comment:
Optional: add a check that the constructor will revert with `ZeroAddress` if the LINK token address is set to zero.

Reply:
Right. I wanted to add that test but I forgot about it. Thanks for catching this.

Reply:
Oh, it's sort of already tested through this [PR](https://github.com/smartcontractkit/chainlink/pull/12483)